cmd/cluster-network-operator/main.go

```diff
@@ -15,6 +15,7 @@ import (
 	k8sutil "github.com/openshift/cluster-network-operator/pkg/util/k8s"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"k8s.io/client-go/tools/clientcmd"
+	_ "k8s.io/code-generator"
```

Comment:
This should probably go in `dependencymagnet`

Reply:
it already is there, will just drop this one